### PR TITLE
Judgement request added

### LINKED ISSUE #38: Add a guide "How to Request Identity Judgment For SORA"
https://docs.google.com/document/d/1AugYAY4IZTNN7agfmx8mhJhnyzwXG-IMW3XA4uupxNU/edit?usp=sharing

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

File: src/id.md [hunk 1]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+- Reg_Index 1 is `SPICY`
+
+- Telegram Username: https://t.me/SpicyHotWings
+
+- Twitter: @mishokanaria
+
+- Riot: @spicyhotwings:matrix.org
+
+4 Verify your identity
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
4. Verify your identity.
```

File: src/id.md [hunk 2]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+## Request Identity Judgment
+
+Once you have finished following all the steps outlined in the [Register an identity](/id.md#register-an-identity) section, you can request an additional verification to obtain a prominent green checkmark next to your account name. This verification serves as an indication that you are a recognized and verified user, allowing others to easily identify and trust your account.
+
+1. Request judgement from a registrar.
+
+On polkadot.js, under the Developer tab, click Extrinsics -> Using the account you wish to be verified, select `Identity` and `requestJudgement` -> Under `regIndex`, place `1` (`reg_index` = `0` is no longer active) -> Registrar `1` has a fee of 1 `XOR` -> Submit the transaction.
+
+<center><img src="/.gitbook/assets/requestJudgement.png" width="600"></center>
```
Comment (outoftardis) [inline src/id.md]: everything that is related to a particular step should be indented, please do this for all steps

File: src/id.md [hunk 3]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+2. Donate to Community Marketing Funds address
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
2. Donate to Community Marketing Funds address.
```

File: src/id.md [hunk 4]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+Donate a minimum of 1 `XOR` per account requesting judgement to the Community Marketing Funds address.
+This is a community base wallet which funds are used for marketing SORA: `cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
Donate a minimum of 1 `XOR` per account requesting judgement to the Community Marketing Funds address (`cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`).
The funds from this community wallet are used for SORA marketing. 
```

maybe something like this

File: src/id.md [hunk 5]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+Direct contact with the registrar is required - the contact info is in their identity:
+
+- Reg_Index 1 is `SPICY`
+
+- Telegram Username: https://t.me/SpicyHotWings
+
+- Twitter: @mishokanaria
+
+- Riot: @spicyhotwings:matrix.org
```
Comment (outoftardis) [inline src/id.md]: this can be a table

File: src/id.md [hunk 6]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+<center><img src="/.gitbook/assets/requestJudgement.png" width="600"></center>
+
+2. Donate to Community Marketing Funds address
+
+Donate a minimum of 1 `XOR` per account requesting judgement to the Community Marketing Funds address.
+This is a community base wallet which funds are used for marketing SORA: `cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`
+
+3. Contact the registrar
+
+Direct contact with the registrar is required - the contact info is in their identity:
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
3. Contact the registrar directly. You can find their contact info in their profile.
```

File: src/id.md [hunk 7]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
+
+- Riot: @spicyhotwings:matrix.org
+
+4 Verify your identity
+
+You will need to prove that the information on-chain is legitimate. Each registrar will have their own set of procedures to verify your identity and values, and only once you've satisfied their requirements will the process continue.
+
+::: info
+
+Changing even a single field's value after you've been verified will un-verify your account and you will need to start the judgement process anew. However, you can still change fields while the judgement is going on - it's up to the registrar to keep an eye on the changes.
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
You can freely change any values during the judgement process until you are verified. However, changing the value of even a single field after you've been verified will un-verify your account and you will need to start the judgement process anew.
```

File: src/id.md [hunk 8]
```diff
@@ -30,3 +28,40 @@ Follow these steps to set up an identity:
...
 
 Once the transaction is completed your account will show an identity!
+
+## Request Identity Judgment
+
+Once you have finished following all the steps outlined in the [Register an identity](/id.md#register-an-identity) section, you can request an additional verification to obtain a prominent green checkmark next to your account name. This verification serves as an indication that you are a recognized and verified user, allowing others to easily identify and trust your account.
+
+1. Request judgement from a registrar.
+
+On polkadot.js, under the Developer tab, click Extrinsics -> Using the account you wish to be verified, select `Identity` and `requestJudgement` -> Under `regIndex`, place `1` (`reg_index` = `0` is no longer active) -> Registrar `1` has a fee of 1 `XOR` -> Submit the transaction.
```
Comment (outoftardis) [inline src/id.md]: this better be separated into steps, I think it'd be more readable

File: src/id.md (hunk 3 above)
Comment (WRRicht3r) [inline src/id.md]:
```suggestion
2. Donate to the Community Marketing Fund's address
```

File: src/id.md (hunk 8 above)
Comment (bgodlin) [reply src/id.md]: Could you please give an example?

File: src/id.md [hunk 9]
```diff
@@ -1,8 +1,6 @@
 # Identity
 
-SORA allows you to store profile information in order to identify and discover people across the network. It is convenient for users to use the identity and registrar feature.
-
-A tutorial is provided below on how to set up an identity for yourself.
+SORA allows you to store profile information in order to identify and discover people across the network. It is convenient for users to use the identity and registrar feature. A tutorial is provided below on how to set up an identity for yourself.
```
Comment (WRRicht3r) [inline src/id.md]:
```suggestion
SORA allows you to store profile information in order to identify and discover people across the network. It's convenient for users to use the identity and registrar feature. A tutorial is provided below on how to set up an identity for yourself.
```

Comment (WRRicht3r) [reply src/id.md]: @outoftardis this is a case where you would use it's :P

File: src/id.md (hunk 8 above)
Comment (WRRicht3r) [inline src/id.md]:
```suggestion
On polkadot.js, under the Developer tab; 
- click Extrinsics -> Using the account you wish to be verified, 
- select `Identity` and `requestJudgement` -> Under `regIndex`, 
- place `1` (`reg_index` = `0` is no longer active) -> Registrar `1` has a fee of 1 `XOR` -> 
- Submit the transaction.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/id.md (hunk 9 above)
Comment (outoftardis) [reply src/id.md]: I mean, those are not the same cases at all...

File: src/id.md (hunk 8 above)
Comment (outoftardis) [reply src/id.md]: something like what William suggested in his review

File: src/id.md (hunk 5 above)
Comment (bgodlin) [reply src/id.md]: https://github.com/sora-xor/sora-docs/pull/79/commits/22bfea0692bd36178fda66e502fec379d99dbe5e

File: src/id.md [hunk 10]
```diff
@@ -1,8 +1,6 @@
 # Identity
 
-SORA allows you to store profile information in order to identify and discover people across the network. It is convenient for users to use the identity and registrar feature.
-
-A tutorial is provided below on how to set up an identity for yourself.
+SORA allows you to store profile information in order to identify and discover people across the network. It's convenient for users to use the identity and registrar feature. A tutorial is provided below on how to set up an identity for yourself.
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
SORA allows you to store profile information in order to identify and discover people across the network. It's convenient for users to use the identity and registrar features. A tutorial is provided below on how to set up an identity for yourself.
```

File: src/id.md [hunk 11]
```diff
@@ -30,3 +28,41 @@ Follow these steps to set up an identity:
...
+1. Request judgement from a registrar.
+
+On polkadot.js, under the Developer tab;
+
+- click Extrinsics -> Using the account you wish to be verified,
+- select `Identity` and `requestJudgement` -> Under `regIndex`,
+- place `1` (`reg_index` = `0` is no longer active) -> Registrar `1` has a fee of 1 `XOR` ->
+- Submit the transaction.
+
+<center><img src="/.gitbook/assets/requestJudgement.png" width="600"></center>
```
Comment (outoftardis) [inline src/id.md]:
```suggestion
1. Request judgement from a registrar on polkadot.js.

  Using the account you wish to verify, go under the Developer tab and follow these steps:

  - Click Extrinsics.
  - Select `Identity` and `requestJudgement`.
  - Under `regIndex`, place `1` (`reg_index` = `0` is no longer active). Registrar `1` has a fee of 1 `XOR`.
  - Submit the transaction.

  <center><img src="/.gitbook/assets/requestJudgement.png" width="600"></center>
```

File: src/id.md [hunk 12]
```diff
@@ -30,3 +28,41 @@ Follow these steps to set up an identity:
...
+
+2. Donate to Community Marketing Funds address.
+
+Donate a minimum of 1 `XOR` per account requesting judgement to the Community Marketing Funds address (`cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`).
+The funds from this community wallet are used for SORA marketing.
```
Comment (outoftardis) [inline src/id.md:50]:
```suggestion
  Donate a minimum of 1 `XOR` per account requesting judgement to the Community Marketing Funds address (`cnSN9X2HNU3uYjRbGvTwpgUdtVDuNk1g29XkcQNoNBQxgt1ii`).
  The funds from this community wallet are used for SORA marketing.
```

File: src/id.md [hunk 13]
```diff
@@ -30,3 +28,41 @@ Follow these steps to set up an identity:
...
+
+3. Contact the registrar directly. You can find their contact info in their profile.
+
+| Social media/messaging platform | Identifier                 |
+| ------------------------------- | -------------------------- |
+| Telegram                        | https://t.me/SpicyHotWings |
+| Twitter                         | @mishokanaria              |
+| Riot                            | @spicyhotwings:matrix.org  |
```
Comment (outoftardis) [inline src/id.md:58]:
```suggestion
  | Social media/messaging platform | Identifier                 |
  | ------------------------------- | -------------------------- |
  | Telegram                        | https://t.me/SpicyHotWings |
  | Twitter                         | @mishokanaria              |
  | Riot                            | @spicyhotwings:matrix.org  |
```

File: src/id.md [hunk 14]
```diff
@@ -30,3 +28,41 @@ Follow these steps to set up an identity:
...
+
+4. Verify your identity.
+
+You will need to prove that the information on-chain is legitimate. Each registrar will have their own set of procedures to verify your identity and values, and only once you've satisfied their requirements will the process continue.
+
+::: info
+
+You can freely change any values during the judgement process until you are verified. However, changing the value of even a single field after you've been verified will un-verify your account and you will need to start the judgement process anew.
+
+:::
```
Comment (outoftardis) [inline src/id.md:68]:
```suggestion
  You will need to prove that the information on-chain is legitimate. Each registrar will have their own set of procedures to verify your identity and values, and only once you've satisfied their requirements will the process continue.

  ::: info

  You can freely change any values during the judgement process until you are verified. However, changing the value of even a single field after you've been verified will un-verify your account and you will need to start the judgement process anew.

  :::
```

**Comment by bgodlin** [comment]
@outoftardis I tried to apply your suggested changes, but seems like they were automatically reverted by https://github.com/sora-xor/sora-docs/pull/79/commits/742bf92840409ef64a9f5a601c5917cb155037a0

**Comment by outoftardis** [comment]
> @outoftardis I tried to apply your suggested changes, but seems like they were automatically reverted by [742bf92](https://github.com/sora-xor/sora-docs/commit/742bf92840409ef64a9f5a601c5917cb155037a0)

hmmm... the prettier configs should be updated then, I guess

**Comment by bgodlin** [comment]
> > @outoftardis I tried to apply your suggested changes, but seems like they were automatically reverted by [742bf92](https://github.com/sora-xor/sora-docs/commit/742bf92840409ef64a9f5a601c5917cb155037a0)
> 
> hmmm... the prettier configs should be updated then, I guess

Why do you think we need to include those indentations on a Markdown level, considering that it is not applied utilising an auto-formatting tool like Prettier? Which official formatting standard are we trying to adhere to in this case?

**Comment by bgodlin** [comment]
@WRRicht3r What additional changes do I need to make in order to meet your review requirements?

**Comment by outoftardis** [comment]
> > > @outoftardis I tried to apply your suggested changes, but seems like they were automatically reverted by [742bf92](https://github.com/sora-xor/sora-docs/commit/742bf92840409ef64a9f5a601c5917cb155037a0)
> > 
> > 
> > hmmm... the prettier configs should be updated then, I guess
> 
> Why do you think we need to include those indentations on a Markdown level, considering that it is not applied utilising an auto-formatting tool like Prettier? Which official formatting standard are we trying to adhere to in this case?

this is just how lists are supposed to work: if you have extra info for one of the items on the list that you want to put in a separate paragraph, that paragraph should be indented so it doesn't break the list. If you don't do it, you basically have multiple lists. Maybe it's not that obvious with numbered lists in Markdown because you put the numbers yourself, but e.g. in restructered text (similar markup to markdown) the list would literally break and the numbered steps would start counting from 1 again. I can't quickly find the source for this rule, but if you for example look at the Microsoft style guide for writing documentation, you'll see this is exactly how lists are written there.

**Comment by bgodlin** [comment]
> > > > @outoftardis I tried to apply your suggested changes, but seems like they were automatically reverted by [742bf92](https://github.com/sora-xor/sora-docs/commit/742bf92840409ef64a9f5a601c5917cb155037a0)
> > > 
> > > 
> > > hmmm... the prettier configs should be updated then, I guess
> > 
> > 
> > Why do you think we need to include those indentations on a Markdown level, considering that it is not applied utilising an auto-formatting tool like Prettier? Which official formatting standard are we trying to adhere to in this case?
> 
> this is just how lists are supposed to work: if you have extra info for one of the items on the list that you want to put in a separate paragraph, that paragraph should be indented so it doesn't break the list. If you don't do it, you basically have multiple lists. Maybe it's not that obvious with numbered lists in Markdown because you put the numbers yourself, but e.g. in restructered text (similar markup to markdown) the list would literally break and the numbered steps would start counting from 1 again. I can't quickly find the source for this rule, but if you for example look at the Microsoft style guide for writing documentation, you'll see this is exactly how lists are written there.

I understand the explanation you provided, and I appreciate the thoroughness. 

However, after trying both with and without the indentations in the preview, I observed that the results were the same. Therefore, in my opinion, disabling the auto-formatter to preserve the indentations may introduce more formatting issues. I propose that we explore options to configure the existing auto-formatter to support the indentations, or alternatively, consider replacing the current auto-formatter with a different one. Nevertheless, I believe these changes can be prioritized lower since they primarily relate to text formatting, and there are other tasks to address first.

**Comment by outoftardis** [comment]
> Nevertheless, I believe these changes can be prioritized lower since they primarily relate to text formatting, and there are other tasks to address first.

Sure

**Review by outoftardis — APPROVED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Pull request merged**